### PR TITLE
Water Property Package Documentation

### PR DESCRIPTION
**Pull request by MarcusHolly**
## Fixes/Resolves:

Issue #895 

## Summary/Motivation:

Adds documentation for the water property package. It is similar to the seawater property package - the only major differences being that this property package is pure H2O and supports liquid and vapor phases

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by agarciadiego — APPROVED** [review]

File: docs/technical_reference/property_models/water.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+   * 1e-5 for the liquid specific enthalpy
+   * 1e-6 for the vapor specific enthalpy
+   * 1e-5 for saturation pressure
+   * 1e-3 for the liquid specific heat capacity
+   * 1e-3 for the vapor specific heat capacity
+   * 1e-6 for latent heat of vaporization
+
+Scaling factors for other variables can be calculated based on their relationships with the user-supplied or default scaling factors.
+   
+Reference
```
Comment (tarnold17) [inline docs/technical_reference/property_models/water.rst]:
```suggestion
References
```

File: docs/technical_reference/property_models/water.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+   "Volumetric flowrate", ":math:`Q = \frac{\sum_{j} M_j}{\rho}`"
+   "Mass concentration", ":math:`C_j = x_j \cdotp \rho`"
+   "Specific enthalpy of liquid", "Equations 25-27 in Nayar et al. (2016)"
+   "Specific enthalpy of vapor", "Equations 25-27 in Nayar et al. (2016) :math:`+ h_{vap}`"
+   "Enthalpy flow", ":math:`H = \sum_{j} M_j \cdotp \widehat{H}`"
+   "Component mole flowrate", ":math:`N_j = \frac{M_j}{MW_j}`"
+   "Component mole fraction", ":math:`y_j = \frac{N_j}{\sum_{j} N_j}`"
+   "Saturation pressure", "Equation 6 in Nayar et al. (2016)"
+   "Specific heat capacity of liquid", "Equation 9 in Sharqawy et al. (2010)"
+   "Specific heat capacity of vapor", "Shomate equation from NIST WebBook"
```
Comment (tarnold17) [inline docs/technical_reference/property_models/water.rst:63]: I think you should consider adding an entry for the NIST WebBook to the list of references at the end of the page.

File: docs/technical_reference/property_models/water.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+   * 1e-3 for the liquid specific heat capacity
+   * 1e-3 for the vapor specific heat capacity
+   * 1e-6 for latent heat of vaporization
+
+Scaling factors for other variables can be calculated based on their relationships with the user-supplied or default scaling factors.
+   
+Reference
+---------
+
+K.G.Nayar, M.H.Sharqawy, L.D.Banchik, and J.H.Lienhard V, "Thermophysical properties of seawater: A review and new correlations that include pressure dependence,"Desalination, Vol.390, pp.1 - 24, 2016. https://doi.org/10.1016/j.desal.2016.02.024
```
Comment (tarnold17) [inline docs/technical_reference/property_models/water.rst]:
```suggestion
K.G. Nayar, M.H. Sharqawy, L.D. Banchik, and J.H. Lienhard V, "Thermophysical properties of seawater: A review and new correlations that include pressure dependence,"Desalination, Vol.390, pp.1 - 24, 2016. https://doi.org/10.1016/j.desal.2016.02.024
```

File: docs/technical_reference/property_models/water.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+M.H. Sharqawy, J.H.L. V, S.M. Zubair, Thermophysical properties of seawater: a review of existing correlations and data, Desalination and Water Treatment. 16 (2010) 354–380. https://doi.org/10.5004/dwt.2010.1079. (2017 corrections provided at http://web.mit.edu/seawater )
```
Comment (tarnold17) [inline docs/technical_reference/property_models/water.rst]:
```suggestion
M.H. Sharqawy, J.H.L. V, S.M. Zubair, Thermophysical properties of seawater: a review of existing correlations and data, Desalination and Water Treatment. 16 (2010) 354–380. https://doi.org/10.5004/dwt.2010.1079. (2017 corrections provided at http://web.mit.edu/seawater)
```

**Review by tarnold17 — APPROVED** [review]
Looks mostly good to me - just some minor formatting suggestions.

**Pull request merged**